src/Orleans.CodeGenerator/WellKnownTypes.cs

```diff
@@ -63,6 +63,7 @@ public WellKnownTypes(Compilation compilation)
             SerializerFeature = Type("Orleans.Serialization.SerializerFeature");
             String = compilation.GetSpecialType(SpecialType.System_String);
             Task = Type("System.Threading.Tasks.Task");
+            ValueTaskNamed = Type("System.Threading.Tasks.ValueTask");
```

Comment:
Why this duplicate entry here?
There is a perfectly good entry two lines below already that doesn't break `netstandard 2.0` compatibility.